cellmlmanip/model.py

```diff
@@ -643,7 +643,13 @@ def get_initial_value(self, symbol):
         :param symbol: Sympy Dummy object of required symbol
         :return: float of initial value
         """
-        return float(self.graph.nodes[symbol]['initial_value'])
+        # fixed for issue where graph.nodes[symbol] does not have an 'initial_value' entry
+        node = self.graph.nodes[symbol]
+        if 'initial_value' in node:
+            return float(node['initial_value'])
+        # no initial_value entry try the dummy_metadata
```

Comment:
Would it be better just to do this approach always? Then we wouldn't need any comments here, just the two lines of code below!

Reply:
My rationale was that the original seemed to work for most things but I've simplified it until we hit a case where that one doesnt work :-)